src/index.ts

```diff
@@ -83,18 +83,21 @@ export interface Options {
 function connect(connectionConfig: pg.ClientConfig | undefined, emitter: TypedEventEmitter<PgListenEvents>, options: Options) {
   connectionLogger("Creating PostgreSQL client for notification streaming")
 
-  const { retryInterval = 500, retryLimit = Infinity, retryTimeout = 3000 } = options
   const effectiveConnectionConfig: pg.ClientConfig = { ...connectionConfig, keepAlive: true }
 
   const Client = options.native && pg.native ? pg.native.Client : pg.Client
   const dbClient = new Client(effectiveConnectionConfig)
-  const getRetryInterval = typeof retryInterval === "function" ? retryInterval : () => retryInterval
```

Comment:
`retryInterval` was not defined at this point.

Reply:
Well, it was defined on L86 with a default value but you removed it. Same applies for `retryLimit` and `retryTimeout`.

Reply:
I was surely missing a coffee or two that day 🤦🏻 . I'll close this PR and prepare a new one.